tests/plan/import/basic.sh

```diff
@@ -102,6 +102,11 @@ rlJournalStart
         rlAssertGrep "pass /lint/tests" $rlRun_LOG
     rlPhaseEnd
 
+    rlPhaseStartTest "Environment is inherited from the importing plan"
+        rlRun -s "tmt run -ddd discover plan --name /plans/environment"
```

Comment:
Does it work if this is run on `/tests/plan/import/modify.sh` with the plan `/importing-other-plan-and-modify-environment` in `/tests/plan/import/modify-data/plans.fmf`?

Reply:
@LecrisUT I am trying to understand what you are on to:

```
❯ tmt run -ddd discover plan --name /importing-other-plan-and-modify-environment |& grep VARIABLE
    VARIABLE: foobar
```

But looks to me it does?

Reply:
The main idea is if we can avoid creating a new tmt plan for this test, and instead edit the beakerlib script to capture the case where it failed (before the fix commit was applied)

Reply:
Now I see @LecrisUT's comment: The newly added plan is duplicating the existing environment modification plan so it makes sense to just simply extend the existing one. @LecrisUT did you mean something like 3d926ff2?

Reply:
maybe, but I do not consider it important enough to look into this more, @happz @psss @lukaszachy wdyt?

Reply:
@thrix, I'm already proposing the simplification in 3d926ff2. Are you ok with it?

Reply:
Indeed 3d926ff27d9ccd22683e3d63dd71c49f334238a8 does the magic thank you for the update :)